mlflow/temporary_db_migrations_for_pre_1_users/versions/ff01da956556_ensure_unique_constraint_names.py

```diff
@@ -177,7 +177,7 @@ def upgrade():
             batch_op.drop_constraint(constraint_name="lifecycle_stage", type_="check")
         batch_op.create_check_constraint(
             constraint_name="experiments_lifecycle_stage",
-            condition=column("lifecycle_stage").in_(["active", "deleted"]),
+            condition=column("lifecycle_stage", type_=String).in_(["active", "deleted"]),
```

Comment:
Why do we need to specify `type` now?

Reply:
According to https://github.com/mlflow/mlflow/blob/master/mlflow/temporary_db_migrations_for_pre_1_users/README, these migrations are obsolete. Happy to file a follow-up ticket to remove them.

Reply:
> Note that this directory will be removed in MLflow 1.1.

Let's remove it :)